### PR TITLE
test: nepali numbers module

### PR DESCRIPTION
**Pull request by golesuman**
## Description

Add tests with full test coverage

## Fixes Issue
- #64 
## Type of Change

Please mark the appropriate option below to describe the type of change your pull request introduces:

- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Documentation update
- [ ] Refactor
- [ ] Other (please specify)

## Checklist

- [x] I have used [semantic commit messages](https://seesparkbox.com/foundry/semantic_commit_messages).
      Examples: `"fix(google): Fixed foobar bug"`, `"feat(accounts): Added foobar feature"`.
- [x] I have used pre-commit hooks.
- [ ] I have added/updated the necessary documentation on `README.md`.
- [ ] I have updated `CHANGELOG.md` for the significant changes.
- [x] I have added appropriate test cases (if applicable) to ensure the changes are functioning correctly.
- [ ] My pull request has a clear title and description.

## Additional Notes

[Add any additional notes or context that you think the reviewers should know about.]

By submitting this pull request, I confirm that I have read and complied with the contribution guidelines of this project.

### CODE REVIEW

**Comment by sugat009** [comment]
Hello @golesuman , thank you for your contribution. 
The tests that you have added might have been misplaced. `TestNumber` class is for the tests of the `number` package as a whole. We try to make sure that the test classes only test one particular entity, so `TestNepaliNumberMethods` class in the same test file might be a better choice to add your tests.
Also, we follow the convention of writing commit messages of the form `<type>: <verb> <what was done>`.
For example in your case it might be `test: Add object representation and equality tests for nepalinumber`.

File: nepali/tests/test_number.py [hunk 1]
```diff
@@ -49,6 +49,18 @@ def test_number_convert_and_add_comma(self):
             NepaliNumber.convert_and_add_comma("123456789"), "१२,३४,५६,७८९"
         )
 
+    def test_object_representation(self):
```
Comment (sugat009) [inline nepali/tests/test_number.py:52]: issue: Tests in wrong class, move to correct class

**Review by sugat009 — CHANGES_REQUESTED** [review]

**Comment by sugat009** [comment]
@golesuman Can you also squash your commit into one commit?

File: nepali/tests/test_number.py [hunk 2]
```diff
@@ -3124,3 +3136,16 @@ def test_nepalinumber_str_ne_for_large_negative_float(self):
     def test_nepalinumber_str_ne_for_zero(self):
         self.assertEqual(self.nepalinumber_integer_0.str_ne(), "०")
         self.assertEqual(self.nepalinumber_float_0_0.str_ne(), "०.०")
+
+    def test_object_representation(self):
```
Comment (sugat009) [inline nepali/tests/test_number.py:3140]: issue: The changes have been copied here but the previous changes have not been removed

**Review by sugat009 — CHANGES_REQUESTED** [review]

**Comment by aj3sh** [comment]
Closing this stale PR.
cc: @sugat009

**Pull request closed without merging**